docs/parameters.rst

```diff
@@ -78,6 +78,20 @@ A type can be made optional with the ``e.optional`` function.
   // or without duration
   const result = await query.run(client, {title: 'The Eternals'});
 
+If you are using the queries generator, add the ``optional`` keyword within a type cast:
```

Comment:
These docs are for the query builder, so I think a better place for this would be in the queries generator docs: https://github.com/edgedb/edgedb-js/blob/master/docs/queries.rst

Feel free to update the example there to use a `with` block and show an optional parameter.

Reply:
How's this? It didn't make as much sense to me to put the same content I suggested in `parameters.rst` in `queries.rst`. So instead I just opted to point to the links you mentioned earlier, since a reference to them here would have been useful to me when I was trying to figure out optional parameters.

I also added a caption and changed the type of the code block for the edgeql example file that was already mentioned to make it stand out a little more.